src/lib/markbind/src/patches/htmlparser2.js

```diff
@@ -261,6 +261,28 @@ Tokenizer.prototype._stateBeforeSpecial = function(c) {
 	this._special = result + 1;
 };
 
+/**
+ * Removes the special state if the special tag was self-closed.
+ */
+Tokenizer.prototype._stateInSelfClosingTag = function(c) {
+	if (c === ">") {
+		this._cbs.onselfclosingtag();
+		this._state = TEXT;
+		this._sectionStart = this._index + 1;
+		if (this._special > 2) {
+			/*
+			 We intentionally fail if the special tag in question is
```

Comment:
Minor: Would really help to document/comment what "fail" imply when you set `this._special = SPECIAL_NONE`.

Reply:
Updated